pallets/schemas/src/tests/other_tests.rs

```diff
@@ -1050,3 +1050,31 @@ fn propose_to_create_schema_name_happy_path() {
 		);
 	})
 }
+
+#[test]
+fn genesis_config_build_genesis_schemas() {
+	let mut t = frame_system::GenesisConfig::<Test>::default().build_storage().unwrap();
+	crate::GenesisConfig::<Test> {
+		initial_schemas: serde_json::from_slice(include_bytes!(
+			"../../../../resources/genesis-schemas.json"
+		))
+		.unwrap(),
+		..Default::default()
+	}
+	.assimilate_storage(&mut t)
+	.unwrap();
+
+	let mut ext: sp_io::TestExternalities = t.into();
+
+	ext.execute_with(|| {
+		System::set_block_number(1);
+		let res = SchemasPallet::get_current_schema_identifier_maximum();
+
+		// We should have at least 10
+		assert!(res >= 10);
+
+		// Check that the first schema exists
+		let res = SchemasPallet::get_schema_by_id(1);
+		assert!(res.is_some());
```

Comment:
Isn't the best test, but as it is pushed this through code tested elsewhere, I figured this was good enough.